### ISSUE TITLE
Frontgrip issues

### ISSUE THREAD

**Issue by Blutze** [toadie2k/NIArms#137]
In `NIArms_Core V1.6.rar    30-Apr-2019 22:44  283M`, `hlc_grip_AFG2` and `hlc_grip_PMVFG` aren't in `nia_rifle_grips_slot`. They are on the repo though, and they do have a public scope now.

None of the grips have custom icons, they all use the UTG bipod one.

The tan SAW grip uses the same translation key as the normal one. Together with the icon and most of the model being hidden in use, it took me a while to notice that wasn't a duplicate.

They don't trigger the gripswitch, even with RHS loaded. Adding `weaponInfoType = "RHS_Gripod2";` (or whichever number) fixes this.

There is an rpt error about the function parameter types, but I think we can safely ignore it. Occurs even with only ACE and RHS.

**Comment by toadie2k** [comment]
Yeah, that's de-scoping is intentional for now. Looking to have an RHS-independent (but complementary?) grip system for them when the either XM8s or SCARs come out, they've been made invisible until that's ready to go because I didn't want to have them just hanging out there. Probably should have done that with the SAW grips, but I'll live it for now.
